force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -54,11 +64,17 @@ public with sharing class FlowControlLoopsCollections {
      * @param num2 The second number.
      * @return The maximum of the two numbers, or null if either number is null.
      */
-    public static Integer findMax(Integer num1, Integer num2) {
-        // Initialize largestNum as null
-        Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+    public static Integer findMax(Integer num1, Integer num2) { 
+        if (num1 == null || num2 == null){
+            return null;
+        } else if (num1 > num2) {
+            return num1;
+        } else if (num2 > num1) {
+            return num2;
+        } else {
+            return num1;
+        }
```

Comment:
Excellent. But, since we are calling `return` inside of each `if` statement, we don't need to say `else if`. They can all be `if`